tools/travis.ps1

```diff
@@ -181,7 +181,11 @@ else
         $packages += Start-PSPackage  @packageParams -Type AppImage
         foreach($package in $packages)
         {
-            if($env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
+            # Publish the packages to the nuget feed if:
+            # 1 - It's a fullBuild (allready checked, for not a PR)
+            # 2 - We have the info to publish (NUGET_KEY and NUGET_URL)
+            # 3 - it's a nupkg file
+            if($isFullBuild -and $env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
```

Comment:
actually, a true merge would change the description of the last commit message too.  Only merging to master using the rebase option would have this possible effect.